Dockerfile

```diff
@@ -97,13 +95,17 @@ RUN yum update -y && \
     git config --global user.email "tfmod442916@users.noreply.github.com" && \
     git config --global user.name "github-actions[bot]" && \
     git config --global --add safe.directory '*'
-RUN mkdir /home/runtimeuser && \
-    chmod -R 777 /home/runtimeuser
-COPY .terraformrc /home/runtimeuser/.terraformrc
+# Create home directory, copy over utilities for xyzenv, terraform cli config, and set permissions
+RUN mkdir ${HOME_DIR}
+COPY .terraformrc ${HOME_DIR}/.terraformrc
 COPY --from=build /go/bin /usr/local/go/bin
-COPY --from=build /src/tfenv /home/runtimeuser/tfenv
-RUN chmod 777 /home/runtimeuser/tfenv && \
-    git clone https://github.com/iamhsa/pkenv.git /pkenv && \
-    cd /pkenv && rm -rf .git && \
+COPY --from=build /src/tfenv ${HOME_DIR}/tfenv
+COPY --from=build /src/pkenv ${HOME_DIR}/pkenv
+RUN mkdir ${HOME_DIR}/tflintenv && \
```

Comment:
I guess this is all one line to reduce image layers?

Reply:
Nope. It's a style error 😊